rpc/client.go

```diff
@@ -138,24 +152,32 @@ func (rc *rpcClient) Address() (common.Address, error) {
 	return rc.nodeAddress, nil
 }
 
+// AuthToken fetches an authentication token from the nitro node
+func (rc *rpcClient) AuthToken() (string, error) {
+	if rc.authToken == "" {
```

Comment:
Could we follow the same pattern as [Address](https://github.com/statechannels/go-nitro/blob/7df7f7ea59f544510b015afdcb2a3564ecb4add6/rpc/client.go#L152C16-L152C16) where `AuthToken` just returns the private variable, and we rely on the constructor to set it?

Reply:
Agreed. Cleaned up in 38a65469.